github-config.yaml

```diff
@@ -419,3 +419,10 @@ orgs:
         members:
           - HumairAK
           - durandom
+      training:
+        description: Manage all training content for users and contributors
+        privacy: closed
+        members:
+          - stefwrite

```

Comment:
The member mentioned , i dont see the person listed a member of operate-first
`https://github.com/operate-first/common/blob/b3325bf886388d6c9282650d2fac9a571317fff1/github-config.yaml#L64-L72`

Reply:
https://github.com/operate-first/common/blob/main/github-config.yaml#L80